url.bs

```diff
@@ -62,12 +62,18 @@ might increase in scope somewhat.
 
 <p>This specification depends on the Infra Standard. [[!INFRA]]
 
-<p>Some terms used in this specification are defined in the
-DOM, Encoding, IDNA, and Web IDL Standards.
-[[!DOM]]
-[[!ENCODING]]
-[[!IDNA]]
-[[!WEBIDL]]
+<p>Some terms used in this specification are defined in the following standards and specifications:
+
+<ul class=brief>
+ <li>DOM Standard [[!DOM]]
+ <li>Encoding Standard [[!ENCODING]]
+ <li>File API [[!FILEAPI]]
+ <li>HTML Standard [[!HTML]]
+ <li>Media Source Extensions [[!MEDIA-SOURCE]]
+ <li>Media Capture and Streams [[!MEDIACAPTURE-STREAMS]]
+ <li>Unicode IDNA Compatibility Processing [[!IDNA]]
+ <li>Web IDL [[!WEBIDL]]
+</ul>
```

Comment:
I wonder, have you found this section useful? I'm never quite sure it adds value given that there's so many ways to discover we use terms from other standards already.

I guess it gives a heads up that some things might be unfamiliar if you haven't read these standards...